src/pages/api/profile/get-profile.ts

```diff
@@ -53,7 +53,10 @@ const handler: Handler = async (request, response) => {
     const { data: profileUserInfo } = await profileClient
       .getProfile()
       .catch((error) => {
-        span.setDisclosedAttribute("is profile user info fetched", false);
+        span.setDisclosedAttribute(
+          "is Connect.Profile user info fetched",
+          false,
+        );
```

Comment:
Maybe we could replace `profile` by `Connect.Profile` in this file's other spans, no ?
cf line 49, 76, 81 and 94